wifi-password.sh

```diff
@@ -94,6 +102,9 @@ fi
 if [ $verbose ]; then
   echo "\033[96m ✓ \"$pwd\" \033[39m"
   echo ""
+elif [ $copy ]; then
+  echo $pwd | pbcopy
+  echo "\033[96m ✓ Password copied to clipboard"
 else
   echo $pwd
-fi
+fi
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5027156/17067683/22a22662-504d-11e6-83df-0e64a1000cc8.png)
http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file